skema/gromet/execution_engine/types/other.py

```diff
@@ -129,3 +129,16 @@ class Range:
 
     def exec(input: int) -> range:
         return range(input)
+
+
+class Call:
```

Comment:
We eventually will want to stop using the primitives in `skema/gromet/execution_engine/types/` since they are artifacts left over from an outdated version of the execution engine. We have a YAML file which I believe should compile the same information in a more approachable way at `skema/gromet/primitive_map.yaml`

How large of a change would this require for the Gromet generation?

This is also something we may want to combine with python_builtins.yaml or create a home directory for these type of files.

Reply:
Oh okay. I had forgotten about that. I think as long as we can use the same interface all that would be needed would be to update what's going on behind the scenes. That is, I just need to change the interface to pull from the correct primitive map file. I don't expect there to be any reason to explicitly change anything in the Gromet generation code itself.